### PR TITLE
UI Component deployment

### PR DESCRIPTION
**Pull request by eladiw**
Deploys the 'latest' release from https://github.com/NDPH-ARTS/mts-trial-ui/releases/tag/latest into the storage account $web blob container

Notes for reviewer:
1. Requires a single .zip file for the release. with tag 'latest'.
2. please double check the credentials are hidden. I have but please double verify
3. Customization is not part of this PR

### CODE REVIEW

**Review by mdp0 — DISMISSED** [review]

File: .github/workflows/terraform-plan-apply.yml [hunk 1]
```diff
@@ -163,4 +165,34 @@ jobs:
...
+      - name: UI component - Get assets zip
+        run: |
+          wget https://github.com/NDPH-ARTS/mts-trial-ui/releases/download/latest/all_assets.zip
+
+      # Unzip & Customzie (replace values)
+      - name: UI component - Unzip and modify
+        run: |
+          unzip all_assets.zip -d ui_component
+
+          issuer="https://login.microsoftonline.com/5d23383f-2acb-448e-8353-4b4573b82276/v2.0"
```
Comment (tamirkamara) [inline .github/workflows/terraform-plan-apply.yml]: I think we should add a tenantId to the config json and get this from there.

File: .github/workflows/terraform-plan-apply.yml [hunk 2]
```diff
@@ -163,4 +165,34 @@ jobs:
...
+          sed -i 's/\$\{\{clientId\}\}"/$ARM_CLIENT_ID/' ui_component/index.html
```
Comment (tamirkamara) [inline .github/workflows/terraform-plan-apply.yml]: I don't think this is right - the clientId used by the UI isn't the same one as the terraform. You should also add this to the config json.

File: .github/workflows/terraform-plan-apply.yml [hunk 3]
```diff
@@ -163,4 +165,34 @@ jobs:
...
+        run: |
+          wget https://github.com/NDPH-ARTS/mts-trial-ui/releases/download/latest/all_assets.zip
+
+      # Unzip & Customzie (replace values)
+      - name: UI component - Unzip and modify
+        run: |
+          unzip all_assets.zip -d ui_component
+
+          issuer="https://login.microsoftonline.com/5d23383f-2acb-448e-8353-4b4573b82276/v2.0"
+          sed -i 's/{{clientId}}"/$ARM_CLIENT_ID/' ui_component/index.html
```
Comment (tamirkamara) [inline .github/workflows/terraform-plan-apply.yml]: This should be different than the one terraform uses... probably get this also from the config json.

**Review by tamirkamara — CHANGES_REQUESTED** [review]

**Comment by tamirkamara** [comment]
@eladiw can you rename this branch to confirm to our standard?

**Pull request closed without merging**